gym_multigrid/envs/fourrooms.py

```diff
@@ -50,9 +50,8 @@ class FourRooms(MultiGridEnv):
 
     def __init__(
```

Comment:
Add docstrings for the class and init method to help users understand how to use the env (same for lavarooms)